### PR TITLE
Implement Black formatter

### PR DESCRIPTION
**Pull request by Sebbo94BY**
This pull request would introduce Black as code-style formatter.

Feel free to close it, if you're not interested. I just thought, that this would somehow improve the code-style and standardize it a bit more. :)

### CODE REVIEW

**Comment by Murgeye** [comment]
Sorry, I am a bit busy at the moment, this will take a bit to review (as well as the changes proposed by #23 ).

File: test_TS3Connection.py [hunk 1]
```diff
@@ -17,39 +17,59 @@ def setUp(self) -> None:
         self.conn = MockTS3Connection()
 
     def test_parse_resp_left_event(self):
-        resp = b"notifyclientleftview cfid=1 ctid=0 reasonid=8 " \
-               b"reasonmsg=Left. clid=1"
+        resp = (
+            b"notifyclientleftview cfid=1 ctid=0 reasonid=8 " b"reasonmsg=Left. clid=1"
```
Comment (Murgeye) [inline test_TS3Connection.py]: This does not seem right, this should be changed into a single byte string.

Comment (Sebbo94BY) [reply test_TS3Connection.py]: The behaviour shouldn't change due to this change.

Both versions should result in the same behaviour as you can see below:

```
>>> resp = b"notifyclientleftview cfid=1 ctid=0 reasonid=8 " \
...                b"reasonmsg=Left. clid=1"
>>> 
>>> print( resp )
b'notifyclientleftview cfid=1 ctid=0 reasonid=8 reasonmsg=Left. clid=1'
>>> 
```

```
>>> resp = (
...             b"notifyclientleftview cfid=1 ctid=0 reasonid=8 " b"reasonmsg=Left. clid=1"
... )
>>> 
>>> print( resp )
b'notifyclientleftview cfid=1 ctid=0 reasonid=8 reasonmsg=Left. clid=1'
>>> 
```

Alternativley, we can also write it like this:

```
>>> resp = b"notifyclientleftview cfid=1 ctid=0 reasonid=8 reasonmsg=Left. clid=1"
>>> 
>>> print( resp )
b'notifyclientleftview cfid=1 ctid=0 reasonid=8 reasonmsg=Left. clid=1'
>>> 
```

**Review by Murgeye — APPROVED** [review]
I have updated the pull request with some fixes regarding my newest commits. Should be good now!

**Comment by Sebbo94BY** [comment]
Nice, thank you! :)

**Pull request merged**